tools/coremark.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python3
+import subprocess
+import re
+import numpy
+
+iter = 10
+res = []
+print("Start Test CoreMark benchmark")
+comp_proc = subprocess.check_output(
+    'build/rv32emu build/coremark.elf', shell=True).decode("utf-8")
```

Comment:
You have to validate the existence of `build/rv32emu` before executing the command `build/rv32emu build/coremark.elf`, otherwise you might run into uncertain states.